go.mod

```diff
@@ -2,6 +2,18 @@ module knative.dev/func
 
 go 1.20
 
+replace (
+	// Use older version since it checks ~/.docker/config.json for credentials as opposed to later versions
+	github.com/containers/image/v5 => github.com/containers/image/v5 v5.23.1
+
+	// Tekton CLI uses newer version without currently used constructor code
+	github.com/hinshun/vt10x => github.com/hinshun/vt10x v0.0.0-20180809195222-d55458df857c
+	// Pin k8s.io dependencies to align with Knative and Tekton needs
+	k8s.io/api => k8s.io/api v0.26.5
+	k8s.io/apimachinery => k8s.io/apimachinery v0.26.5
+	k8s.io/client-go => k8s.io/client-go v0.26.5
```

Comment:
Would the `k8s.io` group jump further ahead without pinning them to aligned version?

Reply:
The k8s.io/client-go dep would be `v1.5.2`. The other two would remain the same -- `v0.26.5`.
Would jump of k8s.io/client-go to `v1.5.2` be a problem? @dsimansk